gh_pages/_source/session1/ros2/3-Creating-a-ROS-Package-and-Node.md

```diff
@@ -134,6 +134,8 @@ Your goal is to create your first ROS node:
    * To use a different logging level, replace INFO in `RCLCPP_INFO` or `RCLCPP_INFO_STREAM` with the appropriate level.
    * Use `RCLCPP_INFO` for printf-style logging, and `RCLCPP_INFO_STREAM` for cout-style logging.
 
+   `rclcpp::spin(node)` instructs the ROS2 [executor instance](https://docs.ros.org/en/rolling/Concepts/About-Executors.html) to tell your node about any activity on subscribed topics or requested service connections. Without it, your callback functions won't ever run. Don't worry if some of this language doesn't yet make sense, we'll cover topics and services in the next sections.
```

Comment:
I feel like we could probably get the value of this line without getting too technical here.  I like the link though.  Maybe something like this?

```suggestion
   `rclcpp::spin(node)` instructs the node to listen ([more technical details here](https://docs.ros.org/en/rolling/Concepts/About-Executors.html)) for any activity on subscribed topics or requested service connections. Without it, your callback functions won't ever run. Don't worry if some of this language doesn't yet make sense, we'll cover topics and services in the next sections.
```